UnityProject/Assets/Scripts/Messages/Client/Interaction/RequestExamineMessage.cs

```diff
@@ -42,10 +42,16 @@ public override IEnumerator Process()
 		// anyone extending IExaminable gets a say in it.
 		// Look for examinables.
 		var examinables = NetworkObject.GetComponents<IExaminable>();
+		string msg = "";
 		foreach (var examinable in examinables)
 		{
 			msg += $"{examinable.Examine()}\n";
 		}
+
+		if (msg.Length > 0)
+		{
+			msg = msg.Substring(0, msg.Length-2);
```

Comment:
what's that for?

Reply:
it'll also give an exception if `msg.Length` is 1

Reply:
It'll never be, but i changed it all the same; it was to remove the newline for the last message, but i wrote it at 3am so it's a bit stupid. It's a for loop now.